src/Core/maps/src/Handlers/Map/MapHandler.Android.cs

```diff
@@ -594,22 +648,31 @@ public void OnMapReady(GoogleMap googleMap)
 
 		void GoogleMap.IOnCameraMoveListener.OnCameraMove()
 		{
-			if (_googleMap == null)
+			if (_googleMap is null)
+			{
 				return;
+			}
 
 			_handler?.UpdateVisibleRegion(_googleMap.CameraPosition.Target);
 		}
 
 		protected override void Dispose(bool disposing)
 		{
-			if (disposing && _handler != null)
+			if (disposing && _handler is not null)
 			{
 				_handler = null;
 				_googleMap = null;
 			}
 
 			base.Dispose(disposing);
 		}
-	}
 
+		public void OnMapsSdkInitialized(MapsInitializer.Renderer p0)
+		{
+			if (p0 != MapsInitializer.Renderer.Latest)
+			{
+				_handler?.MauiContext?.Services.GetService<ILogger<MapHandler>>()?.LogWarning("Not using latest maps renderer!");
+			}
+		}
```

Comment:
Added some logging for people in case, for some reason, this is not the latest renderer?